<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>179</width>
      <height>27</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ConfigFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Config Files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_ConfigFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_ConfigFiles">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Offset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset (Pixel)</string>
    </property>
   </widget>
   <widget class="QSlider" name="sli_Offset">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spi_Screens">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>171</y>
      <width>35</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Screens">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Screens</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_ScreensAuto">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>171</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_ScreensManuel">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>171</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Manuel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spi_Offset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>48</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenSnap Path</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_Path">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Command</string>
    </property>
   </widget>
  </widget>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>rad_ScreensAuto</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setScreensToAuto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rad_ScreensManuel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setScreensToManuel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MainWindow</receiver>
   <slot>execute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MainWindow</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ConfigFiles</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sli_Offset</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>syncOffset(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spi_Offset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>syncOffset(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Path</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFolderPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveAsScript()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setScreensToAuto()</slot>
  <slot>setScreensToManuel()</slot>
  <slot>execute()</slot>
  <slot>cancel()</slot>
  <slot>openFolder()</slot>
  <slot>syncOffset(int)</slot>
  <slot>openFolderPath()</slot>
  <slot>saveAsScript()</slot>
 </slots>
</ui>
